content/en/docs/ops/configuration/multi-revision/gateways/index.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Configuring and managing gateways with multiple revisions
+description: Configuring and managing gateways with multiple revisions.
+weight: 25
+keywords: [kubernetes,upgrading]
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+With a single `IstioOperator` CR, any gateways defined in the CR (including the `istio-ingressgateway` installed in the
+default profile) are upgraded in place, even when the canary control plane method is used. This is
+undesirable - gateways are a critical component that should be upgraded last, when the new version control and data
+planes are verified to be working.
+The recommended way to upgrade gateways is to define and manage them in a separate `IstioOperator` CR, separate from
+the one used to install and manage the control plane.
+
+{{< warning >}}
+To avoid problems with `.` (dot) not being a valid character in some Kubernetes paths, the revision name should not
+include `.` (dots).
+{{< /warning >}}
+
+## Istioctl
+
+This section covers installation and upgrade of separate control plane and gateways, using `istioctl`.
+
+### Installation with `istioctl`
+
+1.  Ensure that the main `IstioOperator` CR has a name and does not install a gateway:
+
+{{< text yaml >}}
+# filename: control-plane.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: `IstioOperator`
+metadata:
+  name: control-plane # REQUIRED
+spec:
+  profile: default
+  components:
+    ingressGateways:
+      - name: istio-ingressgateway
+        # MUST BE DISABLED
+        enabled: false
+{{< /text >}}
+
+1.  Create a separate `IstioOperator` CR for the gateway(s), ensuring that it has a name and has the `empty` profile:
+
+{{< text yaml >}}
+# filename: gateways.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: `IstioOperator`
+metadata:
+  name: gateways # REQUIRED
```

Comment:
nit: how about name it `ingress-gateway`?

Reply:
The reason I used this name is because this CR tracks all the gateways - the pattern users would want is to have both ingress and egress in one file like this.